charts/k8s-monitoring/values.yaml

```diff
@@ -82,6 +82,82 @@ metrics:
   # See https://grafana.com/docs/agent/latest/flow/reference/components/prometheus.relabel/#rule-block
   extraMetricRelabelingRules:
 
+  # Metrics from Grafana Agent
+  agent:
+    # -- Scrape metrics from Grafana Agent
+    enabled: true
+
+    # -- (string) How frequently to scrape metrics from Grafana Agent.
+    # Overrides metrics.scrapeInterval
+    # @default -- 60s
+    scrapeInterval:
+
+    # -- Label matchers used by the Grafana Agent to select Grafana Agent pods
+    labelMatchers:
+      app.kubernetes.io/name: grafana-agent.*
+
+    # -- (string) Rule blocks to be added to the discovery.relabel component for Grafana Agent.
+    # See https://grafana.com/docs/agent/latest/flow/reference/components/discovery.relabel/#rule-block
+    extraRelabelingRules:
+
+    # -- (string) Rule blocks to be added to the prometheus.relabel component for Grafana Agent.
+    # See https://grafana.com/docs/agent/latest/flow/reference/components/prometheus.relabel/#rule-block
+    extraMetricRelabelingRules:
+
+    # Grafana Agent pod settings
+    pod:
+      # -- Name of the metrics port
+      port: http-metrics
+      # -- Does this port use TLS?
+      isTLS: false
```

Comment:
Same as in the river, just remove this and hardcode the `http-metrics` port name. This chart controls the deployment of the Grafana Agent, so we don't need ultimate configurability.